src/qcodes/instrument_drivers/AimTTi/AimTTi_PL601P_channels.py

```diff
@@ -2,4 +2,15 @@
 Deprecated Module for backwards compatibility
 """
 
+import warnings
+
+from qcodes.utils import QCoDeSDeprecationWarning
+
 from ._AimTTi_PL_P import AimTTi, AimTTiChannel, NotKnownModel
+
+warnings.warn(
+    "The `qcodes._drivers.AimTTi.AimTTi_PL601P_channels` module is deprecated. "
```

Comment:
```suggestion
    "The `qcodes.instrument_drivers.AimTTi.AimTTi_PL601P_channels` module is deprecated. "
```
This typo seems to be in all the current deprecations